test/helpers/cons.go

```diff
@@ -20,9 +20,102 @@ import (
 )
 
 var (
-	timeout     time.Duration = 300 //WithTimeout helper translate it to seconds
-	basePath                  = "/vagrant/"
-	networkName               = "cilium-net"
+	timeout time.Duration = 300
+
+	// K8s1VMName is the name of the Kubernetes master node when running
+	// Kubernetes tests.
+	K8s1VMName = fmt.Sprintf("k8s1-%s", GetCurrentK8SEnv())
+
+	// K8s2VMName is the name of the Kubernetes worker node when running
+	// Kubernetes tests.
+	K8s2VMName = fmt.Sprintf("k8s2-%s", GetCurrentK8SEnv())
+)
+
+const (
+	basePath = "/vagrant/"
+
+	// VM / Test suite constants.
+	K8s     = "k8s"
+	K8s1    = "k8s1"
+	K8s2    = "k8s2"
+	Runtime = "runtime"
+
+	Enabled  = "enabled"
+	Disabled = "disabled"
+	Total    = "total"
+
+	// PolicyEnforcement represents the PolicyEnforcement configuration option
+	// for the Cilium agent.
+	PolicyEnforcement = "PolicyEnforcement"
+
+	// PolicyEnforcementDefault represents the default PolicyEnforcement mode
+	// for Cilium.
+	PolicyEnforcementDefault = "default"
+
+	// PolicyEnforcementAlways represents the PolicyEnforcement mode
+	// for Cilium in which traffic is denied by default even when no policy
+	// is imported.
+	PolicyEnforcementAlways = "always"
+
+	// PolicyEnforcementNever represents the PolicyEnforcement mode
+	// for Cilium in which traffic is always allowed even if there is a policy
+	// selecting endpoints.
+	PolicyEnforcementNever = "never"
+
+	// Docker Image names
+
+	// CiliumDockerNetwork is the name of the Docker network which Cilium manages.
+	CiliumDockerNetwork = "cilium-net"
+
+	// NetperfImage is the Docker image used for performance testing
+	NetperfImage = "tgraf/netperf"
+
+	// HttpdImage is the image used for starting an HTTP server.
+	HttpdImage = "cilium/demo-httpd"
+
+	// Names of commonly used containers in tests.
+
+	Httpd1 = "httpd1"
+	Httpd2 = "httpd2"
+	Httpd3 = "httpd3"
+	App1   = "app1"
+	App2   = "app2"
+	App3   = "app3"
+	Client = "client"
+	Server = "server"
+	Host   = "host"
+
+	// Container lifecycle actions.
+	Create = "create"
+	Delete = "delete"
+
+	// IP Address families.
+	IPv4 = "IPv4"
+	IPv6 = "IPv6"
+
+	// Configuration options for endpoints. Copied from endpoint/endpoint.go
+	// TODO: these should be converted into types for use in configuration
+	// functions instead of using basic strings.
```

Comment:
No, importing causes all of the transitive dependencies to be imported, which ends up trying to import CGo and thus requires kernel header dependencies. Because ginkgo runs directly on the host, importing will cause compilation failures for ginkgo on the mac. I can file a ticket to maybe factor out endpoint-related constants to a package, `pkg/endpoint/cons`, and then I'd be able to import that package. What do you think?

Reply:
```
 I can file a ticket to maybe factor out endpoint-related constants to a package, pkg/endpoint/cons, and then I'd be able to import that package. What do you think?
```
Yeah, it seems to be the only option.